tests/modules/smoothxg/main.nf

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { SMOOTHXG } from '../../../modules/smoothxg/main.nf'
+
+workflow test_smoothxg {
```

Comment:
Please add another workflow test using the pangenomics test data: https://github.com/nf-core/test-datasets/blob/modules/data/pangenomics/homo_sapiens/pangenome.seqwish.gfa. Thanks!